plugins/obs-outputs/librtmp/rtmp.c

```diff
@@ -315,23 +315,15 @@ RTMP_TLS_LoadCerts(RTMP *r) {
     CertCloseStore(hCertStore, 0);
 #elif defined(__APPLE__)
     SecKeychainRef keychain_ref;
-    CFMutableDictionaryRef search_settings_ref;
     CFArrayRef result_ref;
 
-    if (SecKeychainOpen("/System/Library/Keychains/SystemRootCertificates.keychain",
+    if (SecKeychainCopyDomainDefault(kSecPreferencesDomainSystem,
```

Comment:
`SecKeychainCopyDomainDefault ` is also deprecated https://developer.apple.com/documentation/security/1401993-seckeychaincopydomaindefault?language=objc